src/main/webapp/app/service/firebase/firebase-gene-review-service.ts

```diff
@@ -115,17 +126,15 @@ export class FirebaseGeneReviewService {
       } else if (isDeleteReview(reviewLevel)) {
         const { firebaseArrayPath, deleteIndex } = extractArrayPath(reviewLevel.valuePath);
         const firebasePath = geneFirebasePath + '/' + firebaseArrayPath;
+        const firebasePathType = getFirebasePathType(firebaseArrayPath + '/' + deleteIndex);
+        if (firebasePathType !== undefined) {
+          const innerMap = itemsToDelete[firebasePathType];
```

Comment:
Is `innerMap` used?

Reply:
I use it on the next line, so I don't have to do `itemsToDelete[firebasePathType][firebasepath]`